modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-a-load-balancer_{context}"]
+= Deploying a TCP load balancer on a workload
+
+{product-title} offers a built-in implementation of network load balancers. This process uses the node IP address as the external IP address for the `LoadBalancer` service config file. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* Access to the cluster as a user with the cluster-admin role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure 
+
+. Verify that your pods are running with the following command:
++
+[source,terminal]
+----
+$ oc get pods -A
+----
+
+. Create a namespace with the following commands. This deploys a test application that verifies the connection.
+//This needs more context of some kind
++ 
+[source,terminal]
+----
+$ NAMESPACE=nginx-lb-test
+----
++
+[source,terminal]
+----
+$ oc create ns $NAMESPACE
+----
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user privileged -z default -n $NAMESPACE
+----
+
+. Deploy three replicas of the test `nginx` application in your namespace.
++
+[source,terminal]
+----
+$ oc apply -n $NAMESPACE -f https://raw.githubusercontent.com/openshift/microshift/main/docs/config/nginx-IP-header.yaml
+----
+
+. You can verify that the three replicas started successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n $NAMESPACE
+----
+
+. Create a `LoadBalencer` service config file for the `nginx` test application with the following commands: 
++
+[source,terminal]
+----
+$ oc create -n $NAMESPACE -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: nginx
+  type: LoadBalancer
+EOF
+----
+
+. To verify that the service file exists and the external IP address is properly assigned, run the following command:
```

Comment:
I think we shall also mention that the external IP address shall be identical to the nodeIP.